PL2/PL2-parser/src/main/java/nl/tudelft/pl2016gr2/parser/controller/GfaReader.java

```diff
@@ -28,14 +29,26 @@ public GfaReader(String filename, int graphsize) {
     this.numNodes = graphsize;
     this.sc = new Scanner(GfaReader.class.getClassLoader().getResourceAsStream(filename));
     originalGraph = new OriginalGraph();
-    FullGfaReader.prepNodes(originalGraph, numNodes);
+    prepNodes(originalGraph, numNodes);
     read();
 
     sc.close();
     sc = null;
   }
 
   /**
+   * Initialize the nodes of a graph.
+   *
+   * @param graph the graph of which to initialize a set amount of node.
```

Comment:
`node -> nodes`
`of -> for` (`of` implies that the nodes are already part of the graph, but must be initialized. However, you are adding new nodes to the graph, so you initialize a node `for` the graph, like a present :) )

Consider rephrasing:
`The graph to initialize a set amount of nodes for`